src/main/java/spark/SparkInstance.java

```diff
@@ -313,6 +313,11 @@ public synchronized void stop() {
             server.stop();
             latch = new CountDownLatch(1);
         }
+        sslStores = null;
```

Comment:
extract a method call resetInstance(), and then do setting all those nulls there. It will improve readability